extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-base/src/main/java/org/apache/guacamole/auth/jdbc/JDBCAuthenticationProviderService.java

```diff
@@ -73,8 +80,22 @@ public ModeledUserContext getUserContext(AuthenticationProvider authenticationPr
 
         // Retrieve user account for already-authenticated user
         ModeledUser user = userService.retrieveUser(authenticationProvider, authenticatedUser);
-        if (user == null)
-            return null;
+        if (user == null) {
```

Comment:
It is true that the documentation of the JDBC auth's `AuthenticationProviderService` does not mention this. I can quickly update that.